clients/venice-push-job/src/main/java/com/linkedin/venice/hadoop/VenicePushJob.java

```diff
@@ -2537,8 +2503,13 @@ protected void setupDefaultJobConf(
       conf.set(KAFKA_INPUT_TOPIC, pushJobSetting.kafkaInputTopic);
       conf.set(KAFKA_INPUT_BROKER_URL, pushJobSetting.kafkaInputBrokerUrl);
       conf.setLong(REPUSH_TTL_IN_HOURS, pushJobSetting.repushTtlInHours);
-      conf.setInt(REPUSH_TTL_POLICY, TTLResolutionPolicy.RT_WRITE_ONLY.getValue()); // only support one policy thus not
-                                                                                    // allow any value passed in.
+      if (pushJobSetting.repushTtlInHours != NOT_SET) {
+        conf.setInt(REPUSH_TTL_POLICY, TTLResolutionPolicy.RT_WRITE_ONLY.getValue()); // only support one policy
```

Comment:
In the meeting, we decided to reduce the scope to only support non-WC stores since for WC stores, only looking at the top-level record timestamp will be a semantic change from the current empty push approach.
Can we put this restriction here?

We could talk to the WC customers to understand the desired behavior later.

Reply:
Ack. Will add this!